docs/Project-Configuration/Stepback-Bisection.md

```diff
@@ -0,0 +1,38 @@
+# Stepback Bisection
+Evergreen's stepback bisection performs stepback by continuously reducing the amount of commits needed to test by half. To enable it, go to the project settings 'general' tab.
+
+## Motivation
+Traditionally, Evergreen performed linear stepback only. This is tasking each previous commit one by one. If stepback bisection is enabled, Evergreen will use bisection instead of linear.
+Evergreen performs linear stepback by default, which takes O(n) tasks at worst to find the offending commit. With bisection, Evergreen performs binary search on the remaining tasks, cutting down the worst time to O(logn).
+
+## Definitions
+'Last Passing Stepback Task' is the last known task that passes.
+'Last Failing Stepback Task' is the task of the recent commit that causes stepback, the failing mainline commit.
+'Next Stepback Task' and 'Previous Stepback Task' helps navigation on Spruce to track the path of stepback.
```

Comment:
Can we describe what they are more clearly, i.e. that they are the next and previous tasks that were activated in the bisection process respectively?